<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="12" passed="7" failed="5" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-01-05T11:09:43Z" name="AllTestsSuite" finished-at="2020-01-05T11:12:04Z" duration-ms="141230">
    <groups>
    </groups>
    <test started-at="2020-01-05T11:09:43Z" name="HGSHOP Test" finished-at="2020-01-05T11:12:04Z" duration-ms="141230">
      <class name="Tests.ChartQuantityTest">
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:09:43Z" name="setupTest" finished-at="2020-01-05T11:09:50Z" duration-ms="6697" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="Add500001ProductQuantity()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:09:50Z" name="Add500001ProductQuantity" finished-at="2020-01-05T11:10:00Z" duration-ms="10766" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at Tests.ChartQuantityTest.Add500001ProductQuantity(ChartQuantityTest.java:156)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add500001ProductQuantity -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:00Z" name="teardownTest" finished-at="2020-01-05T11:10:01Z" duration-ms="715" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:01Z" name="setupTest" finished-at="2020-01-05T11:10:08Z" duration-ms="6661" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="AddSameProductTwoTimes()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:08Z" name="AddSameProductTwoTimes" finished-at="2020-01-05T11:10:15Z" duration-ms="7781" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at Tests.ChartQuantityTest.AddSameProductTwoTimes(ChartQuantityTest.java:175)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AddSameProductTwoTimes -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:15Z" name="teardownTest" finished-at="2020-01-05T11:10:16Z" duration-ms="771" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:16Z" name="setupTest" finished-at="2020-01-05T11:10:26Z" duration-ms="9275" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="ChartQuantityTest_ContinueShopping()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:26Z" name="ChartQuantityTest_ContinueShopping" finished-at="2020-01-05T11:10:28Z" duration-ms="2658" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ChartQuantityTest_ContinueShopping -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:28Z" name="teardownTest" finished-at="2020-01-05T11:10:29Z" duration-ms="1049" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:29Z" name="setupTest" finished-at="2020-01-05T11:10:36Z" duration-ms="6925" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="ChartQuantityTest_ContinueShopping2()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:36Z" name="ChartQuantityTest_ContinueShopping2" finished-at="2020-01-05T11:10:42Z" duration-ms="6014" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [2] but found [3]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [2] but found [3]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at Tests.ChartQuantityTest.ChartQuantityTest_ContinueShopping2(ChartQuantityTest.java:65)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ChartQuantityTest_ContinueShopping2 -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:42Z" name="teardownTest" finished-at="2020-01-05T11:10:43Z" duration-ms="715" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:43Z" name="setupTest" finished-at="2020-01-05T11:10:52Z" duration-ms="9428" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="ChartQuantityTest_ContinueShopping_InsertedNumberOfProducts()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:52Z" name="ChartQuantityTest_ContinueShopping_InsertedNumberOfProducts" finished-at="2020-01-05T11:10:55Z" duration-ms="2732" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [10] but found [1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [10] but found [1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at Tests.ChartQuantityTest.ChartQuantityTest_ContinueShopping_InsertedNumberOfProducts(ChartQuantityTest.java:126)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ChartQuantityTest_ContinueShopping_InsertedNumberOfProducts -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:55Z" name="teardownTest" finished-at="2020-01-05T11:10:56Z" duration-ms="711" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:10:56Z" name="setupTest" finished-at="2020-01-05T11:11:05Z" duration-ms="9113" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="ChartQuantityTest_EndShopping()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:11:05Z" name="ChartQuantityTest_EndShopping" finished-at="2020-01-05T11:11:09Z" duration-ms="3943" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ChartQuantityTest_EndShopping -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:11:09Z" name="teardownTest" finished-at="2020-01-05T11:11:10Z" duration-ms="726" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:11:10Z" name="setupTest" finished-at="2020-01-05T11:11:16Z" duration-ms="6888" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="ChartQuantityTest_EndShopping2()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:11:16Z" name="ChartQuantityTest_EndShopping2" finished-at="2020-01-05T11:11:20Z" duration-ms="3298" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ChartQuantityTest_EndShopping2 -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:11:20Z" name="teardownTest" finished-at="2020-01-05T11:11:20Z" duration-ms="702" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:11:20Z" name="setupTest" finished-at="2020-01-05T11:11:29Z" duration-ms="8613" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="ChartQuantityTest_EndShopping_InsertedNumberOfProducts()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:11:29Z" name="ChartQuantityTest_EndShopping_InsertedNumberOfProducts" finished-at="2020-01-05T11:11:32Z" duration-ms="3403" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [10] but found [1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [10] but found [1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at Tests.ChartQuantityTest.ChartQuantityTest_EndShopping_InsertedNumberOfProducts(ChartQuantityTest.java:111)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ChartQuantityTest_EndShopping_InsertedNumberOfProducts -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.ChartQuantityTest@7219ec67]" started-at="2020-01-05T11:11:32Z" name="teardownTest" finished-at="2020-01-05T11:11:33Z" duration-ms="714" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
      </class> <!-- Tests.ChartQuantityTest -->
      <class name="Tests.LoginTest">
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:11:33Z" name="setupTest" finished-at="2020-01-05T11:11:38Z" duration-ms="5078" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="LoginWithFalseUsername()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:11:38Z" name="LoginWithFalseUsername" finished-at="2020-01-05T11:11:40Z" duration-ms="1321" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithFalseUsername -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:11:40Z" name="teardownTest" finished-at="2020-01-05T11:11:40Z" duration-ms="683" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:11:40Z" name="setupTest" finished-at="2020-01-05T11:11:45Z" duration-ms="5245" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="LoginWithRegisteredUsername()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:11:45Z" name="LoginWithRegisteredUsername" finished-at="2020-01-05T11:11:48Z" duration-ms="3002" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithRegisteredUsername -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:11:48Z" name="teardownTest" finished-at="2020-01-05T11:11:49Z" duration-ms="726" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:11:49Z" name="setupTest" finished-at="2020-01-05T11:11:55Z" duration-ms="5572" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="LoginWithRegisteredUsername_FalsePassword()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:11:55Z" name="LoginWithRegisteredUsername_FalsePassword" finished-at="2020-01-05T11:11:56Z" duration-ms="1283" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginWithRegisteredUsername_FalsePassword -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:11:56Z" name="teardownTest" finished-at="2020-01-05T11:11:57Z" duration-ms="685" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
        <test-method is-config="true" signature="setupTest()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:11:57Z" name="setupTest" finished-at="2020-01-05T11:12:02Z" duration-ms="5395" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupTest -->
        <test-method signature="SqlInjectionTest()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:12:02Z" name="SqlInjectionTest" finished-at="2020-01-05T11:12:03Z" duration-ms="1217" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SqlInjectionTest -->
        <test-method is-config="true" signature="teardownTest()[pri:0, instance:Tests.LoginTest@45018215]" started-at="2020-01-05T11:12:03Z" name="teardownTest" finished-at="2020-01-05T11:12:04Z" duration-ms="683" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardownTest -->
      </class> <!-- Tests.LoginTest -->
    </test> <!-- HGSHOP Test -->
  </suite> <!-- AllTestsSuite -->
</testng-results>
